params/config.go

```diff
@@ -255,6 +259,9 @@ type NodeConfig struct {
 	// NoDiscovery set to true will disable discovery protocol.
 	NoDiscovery bool
 
+	// EnabledDiscoveries defines discovery types that will be used.
```

Comment:
It's a bit hard to figure out allowed values. Maybe there should be a reference where allowed values can be found?

Reply:
removed this field